game_loop_entity.py

```diff
@@ -1,13 +1,16 @@
+from led_interface import LedInterface
+from chess_move import ChessMove
+from lcd_interface import LCDInterface
 class GameLoopEntity():
     def __init__(self):
         self.welcomed = False
-        pass
+        self.lcd_interface = LCDInterface()
 
     def prompt_user_for_input(self):
         if self.welcomed == False:
             self.welcomed = True
-            print("Welcome to Auto Chess")
-        print("Enter initial then final position: ")
+            self.lcd_interface.display("Welcome to Auto Chess","","")
+        self.lcd_interface.display("Enter initial then final position: ")
 
     def gather_user_input(self):
```

Comment:
Right now the `initial` and `final` variables are just values typed into the console when the program is run. Now that we have created the `ChessPosition` and `ChessMove` classes we should use those. 

So instead ask the user for an initial position and then a final position and put those values into `ChessPosition` constructors. Then use those 2 `ChessPositions` we just made to create a `ChessMove`.  And then `return` this `ChessMove`.